scenarios/websocket.benchmarks.yml

```diff
@@ -0,0 +1,24 @@
+﻿# Examples:
+# --config benchmarks.websocket.yml --scenario websocket --variable scenario=echo --profile asp-perf-lin
+
+imports:
+  - https://raw.githubusercontent.com/kondratyev-nv/Benchmarks/main/src/WebsocketClient/websocketclient.yml
+  - https://raw.githubusercontent.com/kondratyev-nv/Benchmarks/main/src/BenchmarksApps/Websocket/websocketServer.yml
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+
+variables:
+    serverPort: 5000
+    serverScheme: ws
+
+scenarios:
+  websocket:
+    application:
+      job: websocketServer
+    warmup:
```

Comment:
Ideally the load generator has a warmup argument such that we only need to run the app once, and it will issue both the warmup and the actual load. Usually the same loop but with a timer that start measuring requests after the warmup delay has passed.